puzzles.yaml

```diff
@@ -0,0 +1,20 @@
+app-id: uk.org.greenend.chiark.sgtatham.puzzles 
+runtime: org.gnome.Platform
+runtime-version: '43'
```

Comment:
44 is out. 43 is EOL in less than  month.

Reply:
Changed